src/os/bluestore/BlueStore.cc

```diff
@@ -8479,8 +8475,8 @@ void BlueStore::_kv_sync_thread()
 	  dout(20) << __func__ << " releasing old bluefs 0x" << std::hex
 		   << p.get_start() << "~" << p.get_len() << std::dec
 		   << dendl;
-	  alloc->release(p.get_start(), p.get_len());
 	}
```

Comment:
can just dout .... << " 0x" << std::hex << bluefs_extents_reclaiming << std::dec ... and drop the loop

Reply:
Thanks for pointing this out!